src/shared/components/Chip/Chip.module.scss

```
@@ -0,0 +1,8 @@
+@use '../../../assets/styles/themes';
```

혹시 scss에서 path alias 안됐던 부분 고쳐주셨지 않나요?? 이렇게 수정 부탁드려요.
```suggestion
@use '@/assets/styles/themes';
```

---

어 이거 안고쳐졌어요

---

이것도 찐막으로 요것도요..ㅎㅎ `@use '~@styles/themes';`